spec/features/service_providers_spec.rb

```diff
@@ -1,4 +1,7 @@
 require 'rails_helper'
+def StripTags(str)
+  ActionController::Base.helpers.strip_tags(str)
+end
```

Comment:
1. method names are usually `snake_case`
2. If we want to scope this helper method to just this file, we should define it inside the `feature` block, otherwise it will "leak" to the global namespace and be in all the other files
3. If we want to make this available to all other files, I'd put it in its own file in `spec/support/` somewhere

Reply:
➕

Reply:
well one way to shorthand it would be include the `Forwardable ` module and define a delegator, but defining a new method is the easiest to read

Assigning method to a variable in Ruby is possible but trickier due to the lack of distinction between method call + variable access